code/src/index.css

```diff
@@ -11,3 +15,32 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+.form-text {
+  width: 100%;
+  height: auto;
+  margin: 8px;
+  font-size: 26px;
+}
+
+.form-button {
+  margin: 8px;
+  font-size: 26px;
+}
+
+.message {
+  margin: 8px;
+  font-size: 26px;
+}
+
```

Comment:
Here is some duplication again. If you want you could combine them 👍
`.form-button, .message {`